test/TestComponents.jl

```diff
@@ -0,0 +1,169 @@
+module TestComponents
```

Comment:
@waralex Might be good to construct these from the component sources, these are prone to change over time as props are added, though I understand they're just for testing purposes.

Reply:
@rpkyle It sounds like "let's break tests at the will of the authors of a third-party package": ( We don't test specific components here (these tests should be on the side of the component developer). We test that the dash works correctly with arbitrary components that meet the General rules for all components. The fact that these components are similar to `html_div` or `html_a` is a matter of convenience, but it does not mean that they are the same components that are contained in the `dash_html_components` package

Reply:
@waralex Given that this is a Dash backend, we are aiming to align component structure with that of https://github.com/plotly/dash-component-boilerplate, whatever that looks like for Julia. Assuming that is true, the content is less important.